### PR TITLE
Edited configuration.md to have proper instructions for leveldb usage

### PR DESCRIPTION
**Pull request by MaazBinMusa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ Then you can run the demo script using:
 ```bash
     mkdir -p docker-volume && docker run -v $PWD/docker-volume:/opt/Desktop \
     -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --shm-size=2g \
-    -it openwpm python3 /opt/OpenWPM/demo.py
```
Comment (vringar) [inline README.md:279]: This change is already in the current master. Can you update your master?

File: docs/Configuration.md [hunk 2]
```diff
@@ -382,4 +382,5 @@ Response body content
     to save only specific types of files, for instance
     `browser_params.save_content = "script"` to save only Javascript
```
Comment (vringar) [inline docs/Configuration.md:383]: Could you update this example instead of adding your own?

File: docs/Configuration.md [hunk 3]
```diff
@@ -382,4 +382,5 @@ Response body content
...
     files. This will lessen the performance impact of this instrumentation
-    when a large number of browsers are used in parallel.
+    when a large number of browsers are used in parallel. An example can be "image,script,imageset"
+- You will also need to import levelDBprovider from openwpm/storage/leveldb.py and instantiate it in the TaskManager in demo.py
```
Comment (vringar) [inline docs/Configuration.md:386]:
```suggestion
- You will also need to import LevelDbProvider from openwpm/storage/leveldb.py and instantiate it in the TaskManager in demo.py
```

**Review by vringar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MaazBinMusa) [reply README.md:279]: you are right. thats on me.

**Pull request merged**